code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/internal/MessagingExtension.java

```diff
@@ -297,24 +299,34 @@ void handlePushToken(final Event event) {
                 getApi());
     }
 
-    void handleTrackingInfo(final Event event, final String datasetId) {
+    /**
+     * Handles the push tracking information from the messaging request content event.
+     * <p>
+     *   The push tracking information is sent to the platform via configured dataset.
+     * @param event {@link Event} containing the push tracking information
+     * @param datasetId A valid {@link String} containing the dataset id
+     */
+    private void handleTrackingInfo(@NonNull final Event event,@NonNull final String datasetId) {
```

Comment:
nit: can you run the android studio code formatter? there are some spacing issues that could be resolved.